components/pd_client/src/client.rs

```diff
@@ -282,6 +283,55 @@ impl fmt::Debug for RpcClient {
 const LEADER_CHANGE_RETRY: usize = 10;
 
 impl PdClient for RpcClient {
+    fn load_global_config(&self, list: Vec<String>) -> PdFuture<HashMap<String, String>> {
+        use kvproto::pdpb::LoadGlobalConfigRequest;
+        let mut req = LoadGlobalConfigRequest::new();
+        req.set_names(list.into());
+        let executor = |client: &Client, req| match client
+            .inner
+            .rl()
+            .client_stub
+            .clone()
+            .load_global_config_async(&req)
+        {
+            Ok(grpc_response) => Box::pin(async move {
+                {
+                    let mut res = HashMap::new();
+                    match grpc_response.await {
+                        Ok(grpc_response) => {
+                            for c in grpc_response.get_items() {
+                                if c.has_error() {
+                                    error!(
+                                        "failed to load global config with key {:?}",
+                                        c.get_error()
+                                    );
+                                } else {
+                                    res.insert(c.get_name().to_owned(), c.get_value().to_owned());
+                                }
+                            }
+                            Ok(res)
+                        }
+                        Err(err) => Err(box_err!("{:?}", err)),
+                    }
+                }
+            }) as PdFuture<_>,
+            Err(err) => Box::pin(async move { Err(box_err!("{:?}", err)) }) as PdFuture<_>,
+        };
+        self.pd_client
+            .request(req, executor, LEADER_CHANGE_RETRY)
+            .execute()
+    }
+
+    fn watch_global_config(
+        &self,
+    ) -> Result<grpcio::ClientSStreamReceiver<pdpb::WatchGlobalConfigResponse>> {
```

Comment:
Are there any tests for these implementations?

I'm concerned about `ClientSStreamReceiver` can work well out of grpcio runtime.